src/index.ts

```diff
@@ -87,11 +87,11 @@ const getAssetFromKV = async (event: FetchEvent, options?: Partial<Options>) =>
   const SUPPORTED_METHODS = ['GET', 'HEAD']
 
   if (!SUPPORTED_METHODS.includes(request.method)) {
-    throw new Error(`${request.method} is not a valid request method`)
+    throw new MethodNotAllowedError(`${request.method} is not a valid request method`)
   }
 
   if (typeof ASSET_NAMESPACE === 'undefined') {
-    throw new Error(`there is no ${ASSET_NAMESPACE} namespace bound to the script`)
+    throw new InternalError(`there is no ${ASSET_NAMESPACE} namespace bound to the script`)
```

Comment:
as it sits now, this will write `there is no undefined namespace bound to the script`

```suggestion
    throw new InternalError(`there is no KV namespace bound to the script.`)
```

Reply:
whoops thought this was fixed